content/css/concepts/position/terms/fixed/fixed.md

```diff
@@ -0,0 +1,148 @@
+---
+Title: 'fixed'
+Description: 'Positions an HTML element of a webpage by taking that element out of the normal flow of the document and pinning it to a specified position where it will remain fixed in place, even while scrolling the page.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Attributes'
+  - 'CSS'
+  - 'Design'
+  - 'Development'
+  - 'Elements'
+  - 'Properties'
+  - 'Programming'
+  - 'Responsive'
+  - 'Static Site'
+  - 'Style'
+  - 'Syntax'
+  - 'UI'
+  - 'UX'
+  - 'Values'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+Positions an [html](https://www.codecademy.com/resources/docs/html) element of a webpage by taking that element out of the normal flow of the document and pinning it to a specified [position](https://www.codecademy.com/resources/docs/css/position) where it will remain **fixed** in place, even while scrolling the page.
+
+## Syntax
+## `position: fixed;`
+
+An element with a **fixed** position will be positioned relative to the viewport or the html element. In other words, it is fixed relative to the document it self. 
+
+>Note: This differs from [absolute](https://www.codecademy.com/resources/docs/css/position/absolute) positioning which is positioned relative to its closest ancestor/parent element whose position is also set to a value of [relative](https://www.codecademy.com/resources/docs/css/position/relative) or **absolute** (a non-static position).
+
+### Bottom, Left, Right, and Top
+As with **absolute** and **relative** positioning, once we have positioned our **fixed** element relative to the viewport, we can manipulate its location using the position properties: 
+
+* [bottom](https://www.codecademy.com/resources/docs/css/position/bottom)
+
+* [left](https://www.codecademy.com/resources/docs/css/position/left)
+
+* [right](https://www.codecademy.com/resources/docs/css/position/right)
+
+* [top](https://www.codecademy.com/resources/docs/css/position/top)
+
+
```

Comment:
This can be removed as in this entry we will speak about `fixed` in CSS position